### ISSUE TITLE
Extension function to enable light navigation bar on Oreo+

### ISSUE THREAD

**Issue by rock3r** [android/android-ktx#139]
This is an extension function I frequently end up writing and using in projects:

```kotlin
@RequiresApi(26)
fun Activity.setNavigationBarStyle() = window.apply {
    decorView.systemUiVisibility = decorView.systemUiVisibility or View.SYSTEM_UI_FLAG_LIGHT_NAVIGATION_BAR
  }
```
(off the top of my head)

The only difference with what I usually end up writing is that this will not handle the API < 26 case (I usually NO-OP in those cases) but instead uses a require API annotation.

**Comment by JakeWharton** [comment]
This seems too hyper specific for my taste because it means adding one function for every system UI flag. Exposing an easier way to enable or disable a system UI flag, however, might be more appropriate.

**Comment by rock3r** [comment]
Yeah you have a point. My need is specifically about the navbar, but there's plenty system UI flags that could benefit from a neater way to toggle them. Trust you will find something more generic :)

**Comment by Edward608** [comment]
A quick though of `Window.setLayoutStable(): Window = this.apply { decorView.systemUiVisibility = decorView.systemUiVisibility or View.SYSTEM_UI_FLAG_LAYOUT_STABLE }`

This approach allows chaining of flag application
